scripts/remove

```diff
@@ -1,29 +1,43 @@
 #!/bin/bash
 
-# Exit on command errors and treat unset variables as an error
-set -u
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
 
-# Source app helpers
 source /usr/share/yunohost/helpers
 
-# Get multi-instances specific variables
+#=================================================
+# LOAD SETTINGS
+#=================================================
+
 app=$YNH_APP_INSTANCE_NAME
 
-# Retrieve arguments
-domain=$(ynh_app_setting_get "$app" domain)
-path=$(ynh_app_setting_get "$app" path)
+domain=$(ynh_app_setting_get $app domain)
+port=$(ynh_app_setting_get $app port)
+
+#=================================================
+# STANDARD REMOVE
+#=================================================
+# REMOVE SERVICE FROM ADMIN PANEL
+#=================================================
+
+if yunohost service status | grep -q $app
```

Comment:
Maybe is it better to add the service in the upgrade script (I think there's no error if the service already exists), so we don't have this case to handle?

Reply:
Hum, yes it's not in the upgrade script, but I'm not sure we really have to. The old install script has already the service added in the admin panel.

This verification is more usefull in case of break down during the install script, just to avoid a useless error message.